src/components/ColorPallet/ColorPallet.jsx

```diff
@@ -0,0 +1,56 @@
+import { useData } from "../../context/DataProvider"
+import { updateNote } from "../../services";
+
+const colorPalletData = [
+    {
+        id: 1,
+        cssName: "red",
+        bgProp: "bgRed"
+    },
+    {
+        id: 2,
+        cssName: "green",
+        bgProp: "bgGreen"
+    },
+    {
+        id: 3,
+        cssName: "yellow",
+        bgProp: "bgYellow"
+    },
+    {
+        id: 4,
+        cssName: "purple",
+        bgProp: "bgPurple"
+    },
+    {
+        id: 5,
+        cssName: "naviBlue",
+        bgProp: "bgBlue"
```

Comment:
Why you made this and how you are using it? I am not getting this part. Please explain me.

Reply:
I'm using this array for classname based on that I'm applying colors to the notes